tests/mock_connection.py

```diff
@@ -29,6 +33,13 @@ def __init__(self, user=None, pwd=None, proxies=None, verbose=False, ignore_ssl_
         self.token = None
         self.token_type = None
 
+        # We assign the uri root to a member variable up front, and use that everywhere to
+        # simplify. In order to use an alternate uri root, it must be specified in the constructor.
+        if server_url is not None:
```

Comment:
It _is_ possible to query API's using HTTP, and HTTPS isn't required.  In talking to the software guys, we would like to change this in the future.  But the current state is that it users could connect to an http:// endpoint, so I think the uri_root should strip out that prefix to be more agnostic .

Reply:
This is just on the mock connection

Plus all default endpoints are HTTPS by default

https://github.com/ge-flight-analytics/emspy/blob/e2e3837ce2b96985cb7c60eb2a7030df46e7a32b/emspy/common.py#L4-L10

Reply:
Ah, I missed that this was a mock.  All defaults are https, but users are able to pass their own server_urls.

Reply:
I'm fine not stripping the https:// prefix.  Probably won't cause issues.